custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/util/MavenHelper.java

```diff
@@ -70,7 +70,7 @@ public int run(File buildDir, boolean failOnError, String ... args) throws IOExc
     }
 
     private static String getOsSpecificMavenCommand() {
-        String mvnCmd = "mvn";
+        String mvnCmd = "/home/fcojfernandez/Desarrollo/apache-maven-3.5.4/bin/mvn";
```

Comment:
Wrong commit I guess.

Reply:
Yes, my IDE is not honoring the PATH variable and I have to trick the class. I forgot to revert my local change